src/vm/readytoruninfo.cpp

```diff
@@ -345,7 +345,7 @@ PTR_BYTE ReadyToRunInfo::GetDebugInfo(PTR_RUNTIME_FUNCTION pRuntimeFunction)
 
 BOOL ReadyToRunInfo::IsReadyToRunEnabled()
 {
-    STANDARD_VM_CONTRACT;
+    WRAPPER_NO_CONTRACT;
```

Comment:
This is the change. `STANDARD_VM_CONTRACT` implies `GC_TRIGGERS`, causing a violation when it is called by `EEConfig::sync`. Since `ReadyToRunInfo::IsReadyToRunEnabled` can't really trigger a GC, and is a simple wrapper of `ConfigDWORD.val`, changing the contract to `WRAPPER_NO_CONTRACT` appears to be the easiest solution.